guides/get_action_context.md

```diff
@@ -0,0 +1,50 @@
+# Get action context information 
+
+The App needs access to the ActionContext via ActionToken to provide corresponding configuration page with the specific action context.
```

Comment:
_should be_

If you have registered a Client action handler callback for your App, then when your App is called through this callback, it will receive an client action handler context token. Your App needs use this token with this API to extract the proper action context.

Reply:
Modified as suggested. Thanks!